docs/source-2.0/guides/smithy-cli/cli_installation.rst

```diff
@@ -0,0 +1,470 @@
+        tarball with gpg.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ gpg --verify smithy-install/|linux-tar|.asc smithy-install/|linux-tar|
+
+
+    .. tab:: Manual (ARM)
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ curl -L '|release-uri|/|linux-arm-tar|.{asc,sha256}' -o 'smithy-install/|linux-arm-tar|.#1'
+
+        Verify the checksum of the tarball using the `sha256sum` utility.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ (cd smithy-install && sha256sum -c |linux-arm-tar|.sha256)
+
+        Now, retrieve the public PGP key from the `release`_, and import
+        it into your key-chain.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ curl -L |release-uri|/smithy.asc -o smithy-install/smithy.asc && gpg --import smithy-install/smithy.asc
+
+        Finally, after importing the key, verify the signature of the
+        tarball with gpg.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ gpg --verify smithy-install/|linux-arm-tar|.asc smithy-install/|linux-arm-tar|
+
+
+.. tab:: Windows
+
+    Download the detached signature file (``.asc``) and the sha256-checksum
+    file (``.sha256``) corresponding to the tarball (which you already
+    downloaded) from the `Smithy GitHub releases`_.
+
+    .. tab:: Manual (x64)
+
+        .. code-block:: powershell
+            :caption: powershell
+            :substitutions:
+
+            PS > (wget |release-uri|/|windows-tar|.asc -outfile smithy-install\|windows-tar|.asc) -and
+                (wget |release-uri|/|windows-tar|.sha256 -outfile smithy-install\|windows-tar|.sha256)
+
+        Compute the actual checksum of the tarball using ``certutil``.
+
+        .. code-block:: powershell
+            :caption: powershell
+            :substitutions:
+
+            PS > certutil -hashfile smithy-install\|windows-tar|.sha256 SHA256
+
+        Now, print out the expected checksum from the file that you
+        downloaded (``.sha256``).
+
+        .. code-block:: powershell
+            :caption: powershell
+            :substitutions:
+
+            PS > cat smithy-install\|windows-tar|.sha256
+
+        Verify that the output from the two commands matches. There may be a
+        file-path appended to the latter command output - you
+        may safely ignore that.
+
+        Now, retrieve the public PGP key from the `release`_, and import it into your key-chain.
+
+        .. code-block:: powershell
+            :caption: powershell
+            :substitutions:
+
+            PS > (wget |release-uri|/smithy.asc -outfile smithy-install\smithy.asc) -and (gpg --import smithy-install\smithy.asc)
+
+        Finally, after importing the key, verify the signature of the
+        tarball with gpg.
```

Comment:
Do we have any troubleshooting tips for customers? 
For example if they run into:  https://github.com/awslabs/smithy/blob/main/smithy-cli/config/install#L126

Reply:
I'll add a blurb about it